example/datasets/image_net/example.py

```diff
@@ -0,0 +1,37 @@
+import io
+from pprint import pprint
+
+from PIL import Image as PILImage
+from PIL import ImageDraw
+
+from starwhale import URI, URIType, get_data_loader
+
+
+def draw_bbox(img, bbox_view_):
+    bbox1 = ImageDraw.Draw(img)
+    x, y, w, h = bbox_view_.x, bbox_view_.y, bbox_view_.width, bbox_view_.height
+    bbox1.rectangle(
+        [
+            (x, y),
+            (
+                x + w,
+                y + h,
+            ),
+        ],
+        fill=None,
+        outline="red",
+    )
+
+
+def show():
+    uri = URI("image-net/version/latest", expected_type=URIType.DATASET)
```

Comment:
use starwhale dataset sdk to refactor example?

some example code:

```
from starwhale import dataset
ds = dataset("image-net/version/latest")
row = ds.fetch_one()
...
```

Reply:
done